Lab 7.ipynb

```diff
@@ -0,0 +1,854 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "n_higgs = 100 / 100000\n",
+    "n_qcd = 20000 / 100000\n",
+    "\n",
+    "fig, ax = plt.subplots(1, 1, figsize = (15, 10))\n",
+    "\n",
+    "qcd_weight = np.array([n_qcd] * len(qcd['mass']))\n",
+    "higgs_weight = np.array([n_higgs] * len(higgs['mass']))\n",
+    "\n",
+    "ax.hist([qcd['mass'], higgs['mass']], bins = 100, density = True, stacked = True, weights = [qcd_weight, higgs_weight])\n",
+    "\n",
+    "ax.legend([\"QCD\", \"Higgs\"])\n",
+    "ax.set_title(\"Stacked Histogram of Mass for QCD and Higgs\")\n",
+    "ax.set_xlabel(\"Mass\")\n",
+    "ax.set_ylabel(\"Probability\")\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "c4ec1478-337c-4c7f-b74c-96266135cbcf",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The expected significance of the Higgs Boson is 0.9105386773865511. The approximation is 0.00223606797749979.\n"
+     ]
+    }
+   ],
+   "source": [
+    "n_higgs = 100 / 100000\n",
+    "n_qcd = 20000 / 100000\n",
+    "\n",
+    "exp_prob = stats.poisson.cdf(n_higgs + n_qcd, n_qcd, loc = 0)\n",
+    "exp_sig = stats.norm.ppf(exp_prob)\n",
+    "\n",
+    "approx_sig = n_higgs / np.sqrt(n_qcd)\n",
+    "\n",
+    "print(\"The expected significance of the Higgs Boson is \" + str(exp_sig) + \". The approximation is \" + str(approx_sig) + \".\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c5ebd6cf-1e23-4819-80b9-033e26df45ea",
+   "metadata": {},
+   "source": [
+    "# Cuts"
```

Comment:
Explanation of what this section is and the thought process behind the various cuts